src/main/java/com/project2/telemedicineapi/controllers/DoctorController.java

```diff
@@ -0,0 +1,57 @@
+package com.project2.telemedicineapi.controllers;
+
+import com.project2.telemedicineapi.dto.LoginDTO;
+import com.project2.telemedicineapi.entities.Doctor;
+import com.project2.telemedicineapi.exception.UnAuthorizedResponse;
+import com.project2.telemedicineapi.services.DoctorService;
+import com.project2.telemedicineapi.services.JwtService;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jws;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+//@EnableAutoConfiguration(exclude={DataSourceAutoConfiguration.class, HibernateJpaAutoConfiguration.class})
+@RestController
+@RequestMapping("/doctor")
+public class DoctorController {
+    @Autowired
+    DoctorService doctorService;
+
+    @Autowired
+    JwtService jwtService;
+
+    @PostMapping("/login")
+    public ResponseEntity<?> login(@RequestBody LoginDTO dto) {
+
```

Comment:
Please Wrap this with try and catch to return not found or bad request instead of causing Internal error
![image](https://user-images.githubusercontent.com/46400501/168218064-79b31357-1213-4055-b16b-99dfa61f038f.png)